release-conf.yaml

```diff
@@ -6,3 +6,6 @@ trigger_on_issue: true
 labels:
   - bot
   - release-bot
+
+pypi: true
+pypi_project: packitos
```

Comment:
not necessary, release-bot should take it from `setup.cfg`

Reply:
so why it did not release to pypi?

Reply:
good question, we have no logs from that time (the pod was probably restarted a few hours after the release)

Reply:
ok, I'll drop the changes then